SabreTools.Serialization/Wrappers/PortableExecutable.Extraction.cs

```diff
@@ -176,7 +176,7 @@ public bool ExtractInstallShieldExecutable(string outputDirectory, bool includeD
                 if (overlayAddress < 0)
                     return false;
 
-                const int chunkSize = 64 * 1024;
+                const int chunkSize = 2048 * 1024; //Arbitrary buffer size
```

Comment:
Please remove the comment

Reply:
Alright. I wanted to leave it there so I remember that there isn't a need for this specific size, but fair enough.